lib/language-server.ts

```diff
@@ -110,6 +111,13 @@ connection.onInitialize((params: InitializeParams) => {
         prepareProvider: true
       },
       workspaceSymbolProvider: true,
+      signatureHelpProvider: {
+        triggerCharacters: [
+          ...Array.from(Array(26)).map((e, i) => String.fromCharCode(i + 65)),
+          ...Array.from(Array(26)).map((e, i) => String.fromCharCode(i + 65).toLowerCase()),
+          ...Array.from(Array(10).keys()).map(a => String(a))
```

Comment:
This seems wrong. What is the idea of the triggerChars?

Reply:
Changed it to ( now. Makes more sense tbh (I also just learned the hotkey to trigger this)